docs/architecture.md

```diff
@@ -1,103 +1,172 @@
 # OpenVoiceOS Architecture
-**Editors Note**
-Most of the text in the entire architecture section will not remain, it's my notes so I can create some architecture drawings and simplified information.
+**This section can be a bit technical, but is included for refrence**
 
-## Bus Service
+OVOS is a collection of modular services that work together to provide a seamless, private, open source voice assistant.  Every OVOS service runs as an indivudually and interacts with one another.
+
+The suggested way to start OVOS is with systemd service files.  Most of the supported images run these services as a normal `user` instead of system wide.  If you get and error when using the system files, try using it as a system service.
```

Comment:
```suggestion
The suggested way to start OVOS is with systemd service files.  Most of the supported images run these services as a normal `user` instead of system wide.  If you get an error when using the system files, try using it as a system service.
```